### ISSUE TITLE
left to right square hover  is not working

### ISSUE THREAD

**Issue by gamersudhar** [gudh/ihover#19]
html
<!doctype html>
<html>
<head>
<title>student</title>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="HandheldFriendly" content="true">
<link rel="stylesheet" type="text/css" href="style.css">
<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>

 
  <link rel="stylesheet" type="text/css" href="engine1/style.css" />
  <script type="text/javascript" src="engine1/jquery.js"></script>
  

<script type="text/javascript" src="engine1/wowslider.js"></script>

<script type="text/javascript" src="engine1/script.js"></script>

   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
   <link href="styles/ihover.css" rel="stylesheet">
  <link href="styles/ihover.min.css" rel="stylesheet">

</head>
<body>

<div id="wrapper">
<div id="header">
<div class="company">
<img src="../sudhar/photo/logo.jpg"  height="48px" width="120px" />
</div>
<div class="navbar">
<ul>
<li><a class="active" href="home.html"/>Home</li>
<li><a href="about.html"/>About us</li>
<li><a href="services.html"/>Services</li>
<li><a href="products.html"/>Products</li>
<li><a href="contact.html"/>Contact us</li>
</ul>
</div>
</div>

<div class="container">
<div class="slice">
 
<div id="wowslider-container1">
<div class="ws_images"><ul>
    <li><img src="data1/images/as.jpg" alt="as" title="About us" id="wows1_0"/></li>
    <li><img src="data1/images/c.jpg" alt="c" title="Contact" id="wows1_1"/></li>
    <li><img src="data1/images/h.jpg" alt="h" title="Home" id="wows1_2"/></li>
    <li><img src="data1/images/s.jpg" alt="s" title="s" id="wows1_4"/></li>
  </ul></div>
  <div class="ws_bullets"><div>
    <a href="#" title="as"><span><img src="data1/tooltips/as.jpg" alt="as"/>1</span></a>
    <a href="#" title="c"><span><img src="data1/tooltips/c.jpg" alt="c"/>2</span></a>
    <a href="#" title="h"><span><img src="data1/tooltips/h.jpg" alt="h"/>3</span></a>
    <a href="#" title="p"><span><img src="data1/tooltips/p.jpg" alt="p"/>4</span></a>
    <a href="#" title="s"><span><img src="data1/tooltips/s.jpg" alt="s"/>5</span></a>
  </div></div><div class="ws_script" style="position:absolute;left:-99%"><a href="http://wowslider.com">http://wowslider.com/</a> by WOWSlider.com v8.7</div>
<div class="ws_shadow"></div>
</div>  

</div>
</div>

<div class="container-fluid">
  <div class="row">
  <div class="col-xs-6 col-md-4">
  <div class="thumbnail">
    
    <div class="hovereffect">
        <img class="img-responsive" src="../sudhar/photo/i.jpg" alt="innovation" height="289px" width="289px"/>
        <div class="overlay">
            <h2>Innovation</h2>
        </div>
    </div>

   </div></div>

<div class="col-xs-6 col-md-4">
<div class="thumbnail">
 <div class="hovereffect">
        <img class="img-responsive" src="../sudhar/photo/cr.png" alt="creativity" height="289px" width="289px"/>
        <div class="overlay">
            <h2>Creativity</h2>
        </div>
    </div>

 </div> </div>

<div class="col-xs-6 col-md-4">
<div class="thumbnail">
  <div class="hovereffect">
        <img class="img-responsive" src="../sudhar/photo/po.jpg" alt="Positivity" height="289px" width="289px"/>
        <div class="overlay">
            <h2>Positivity</h2>
        </div>
    </div>

 </div></div>   
 </div>

 
 

<div class="row">
  <div class="col-sm-6">
 
    
    <div class="ih-item square effect15 left_to_right"><a href="#">
        <div class="img"><img src="../sudhar/photo/taj.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
  <div class="col-sm-6">
 
    
    <div class="ih-item square colored effect15 left_to_right"><a href="#">
        <div class="img"><img src="images/assets/rect/2.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
</div>

<div class="row">
  <div class="col-sm-6">
 
    
    <div class="ih-item square effect15 right_to_left"><a href="#">
        <div class="img"><img src="images/assets/rect/3.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
  <div class="col-sm-6">
 
    
    <div class="ih-item square colored effect15 right_to_left"><a href="#">
        <div class="img"><img src="images/assets/rect/4.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
</div>

<div class="row">
  <div class="col-sm-6">
 
    
    <div class="ih-item square effect15 top_to_bottom"><a href="#">
        <div class="img"><img src="images/assets/rect/5.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
  <div class="col-sm-6">
 
    
    <div class="ih-item square colored effect15 top_to_bottom"><a href="#">
        <div class="img"><img src="images/assets/rect/6.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
</div>

<div class="row">
  <div class="col-sm-6">
 
    
    <div class="ih-item square effect15 bottom_to_top"><a href="#">
        <div class="img"><img src="images/assets/rect/7.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
  <div class="col-sm-6">
 
    
    <div class="ih-item square colored effect15 bottom_to_top"><a href="#">
        <div class="img"><img src="images/assets/rect/8.jpg" alt="img"></div>
        <div class="info">
          <h3>Heading here</h3>
          <p>Description goes here</p>
        </div></a></div>
    
 
  </div>
</div>

<div id="con">
<div class="contact">

<h1>Address</h1>

<p><span class="glyphicon glyphicon-map-marker"></span>
Abc private limited
no54,2ndstreet,
madipakkam,
chennai-67.
</p>

 <p><span class="glyphicon glyphicon-envelope"></span> ABCPVT@gmail.com</p>
</div>
<div class="enquiry">

<h1>Enquiry Form</h1>

<table border="0" align="center">
<form name="form" action="email.php" id="form" method="post">
<tr>
<tr>
<td>Name</td>
<td><input name="name" placeholder="Name" type="text" value=""></td>
</tr>

<<tr>
<td>Email</td>
<td><input name="email" placeholder="Email" type="text" value=""></td>
</tr>

<tr>
<td>Phone</td>
<td><input name="phone" placeholder="phonenumber" type="text" value=""></td>
</tr>

<tr>
<td>Message</td>
<td><textarea name="msg" placeholder="Type your text here..."></textarea></td>
</tr>
<td colspan="2" style="text-align:center"><input id="send" name="submit" type="submit" value="Submit"></td>
</tr>
</form>
</table>

</div>
</div>
</div>
<div id="footer">
Copyrights &copy; to WWW.abcpvt.com
<img src="../sudhar/photo/fb.png" alt="fb"/>
<img src="../sudhar/photo/ln.png" alt="tweet"/>
<img src="../sudhar/photo/tw.png" alt="linkedin"/>
<img src="../sudhar/photo/pin.jpg" alt="pinterest" height="40px" width="40px"/>
</div>

</body>
</html>

css
body
{
margin:0px;
}
# wrapper

{
width:100%;
height:auto;  
margin:0px;
background-color:#E9E581;
}
h1 {
    position: relative;
    font-size: 30px;
    z-index: 1;
    overflow: hidden;
    text-align: center;
}
h1:before, h1:after {
    position: absolute;
    top: 51%;
    overflow: hidden;
    width:50%;
    height: 1px;
    content: '\a0';
    background-color:black;
}
h1:before {
    margin-left: -50%;
    text-align: right;
}
.color {
    background-color: #ccc;
}
# header

{
width:100%;
height:70px;
position:fixed;
background-color:black;
right:0px;
z-index: 100;
}
# header

{
top:0;  
}
# footer

{
left:0;
right:0;
z-index: :100;
width:100%;
height:48px;
background-color:black;
color:white;
text-align: center;
}
# footer img

{
float:right;
margin:5px;
padding:5px;
} 

.company
{
  width:50%;
  height:60px;
  color:white;
  float:left;
  text-align:center;
  padding:15px;
}
.navbar
{
height:60px;
width:50%
background-color:black;
color:white;
float:left;
overflow:hidden;

}

 ul li
{
display: block;
position: relative;
padding:10px 15px;
float:left;
}
ul li a
{
color:white;
}
.navbar a:hover
{
    border-bottom:3px solid red;
  padding:18px;

}
# myCarousel

{
background-color:white;
height:370px;
width:100%;
float:left;
}
.carousel-inner > .item > img,
  .carousel-inner > .item > a > img 
  {
      width: 70%;
      height:15%;
      margin: auto;
      background-color:white;
  }

.thumbnail
{
height:300px;
width:300px;
padding:25px auto;
margin:5px;

}
# con

{
width:100%;
height:300px;
}
.contact
{
width:60%;
height:300px;
background-color:grey;
padding:100px;
float:left;
}

.enquiry
{
width:40%;
height:300px;
background-color:seagreen;
float:left;

}
input[type=text]:focus {
    border:3px solid grey;
}
input[type="text"] 
{
    margin: 0 0 15px 0;
}

@media screen and (max-width:959px)
{
# wrapper

{
  width:auto;
}
.navbar
{
  width:auto;
}
img
{
  width:auto;
}

}
@media screen and (max-width:640px)
{
  #wrapper,#con,#enquiry
  {
    width:auto;
  }
}

.hovereffect {
  width: 100%;
  height: 100%;
  float: left;
  overflow: hidden;
  position: relative;
  text-align: center;
  cursor: default;
}

.hovereffect .overlay {
  width: 100%;
  height: 100%;
  position: absolute;
  overflow: hidden;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.6);
  opacity: 0;
  filter: alpha(opacity=0);
  -webkit-transform: translate(460px, -100px) rotate(180deg);
  -ms-transform: translate(460px, -100px) rotate(180deg);
  transform: translate(460px, -100px) rotate(180deg);
  -webkit-transition: all 0.2s 0.4s ease-in-out;
  transition: all 0.2s 0.4s ease-in-out;
}

.hovereffect img {
  display: block;
  position: relative;
  -webkit-transition: all 0.2s ease-in;
  transition: all 0.2s ease-in;
}

.hovereffect h2 {
  text-transform: uppercase;
  color: #fff;
  text-align: center;
  position: relative;
  font-size: 17px;
  padding: 10px;
  background: rgba(0, 0, 0, 0.6);
}

.hovereffect a.info {
  display: inline-block;
  text-decoration: none;
  padding: 7px 14px;
  text-transform: uppercase;
  color: #fff;
  border: 1px solid #fff;
  margin: 50px 0 0 0;
  background-color: transparent;
  -webkit-transform: translateY(-200px);
  -ms-transform: translateY(-200px);
  transform: translateY(-200px);
  -webkit-transition: all 0.2s ease-in-out;
  transition: all 0.2s ease-in-out;
}

.hovereffect a.info:hover {
  box-shadow: 0 0 5px #fff;
}

.hovereffect:hover .overlay {
  opacity: 1;
  filter: alpha(opacity=100);
  -webkit-transition-delay: 0s;
  transition-delay: 0s;
  -webkit-transform: translate(0px, 0px);
  -ms-transform: translate(0px, 0px);
  transform: translate(0px, 0px);
}

.hovereffect:hover h2 {
  -webkit-transform: translateY(0px);
  -ms-transform: translateY(0px);
  transform: translateY(0px);
  -webkit-transition-delay: 0.5s;
  transition-delay: 0.5s;
}

.hovereffect:hover a.info {
  -webkit-transform: translateY(0px);
  -ms-transform: translateY(0px);
  transform: translateY(0px);
  -webkit-transition-delay: 0.3s;
  transition-delay: 0.3s;
}
